cmd/tracee-ebpf/capabilities.go

```diff
@@ -20,20 +20,22 @@ type IKernelVersionInfo interface {
 
 // ensureCapabilities makes sure program runs with required capabilities only
 func ensureCapabilities(OSInfo IKernelVersionInfo, cfg *tracee.Config, allowHighCapabilities bool) error {
```

Comment:
nit: Not sure why you prefixed the `I` in the interface, it's very much just a Java/C# thing. I'd take the opportunity to just rename into `KernelVersionInfo`.